### PR TITLE
feat:make Time2String as common function

### PR DESCRIPTION
**Pull request by liu-song**
**Please provide issue(s) of this PR:**
Fixes #https://github.com/polarismesh/polaris/issues/195

**To help us figure out who should review this PR, please put an X in all the areas that this PR affects.**

- [ ] Configuration
- [ ] Docs
- [ ] Installation
- [ ] Performance and Scalability
- [ ] Naming
- [ ] HealthCheck
- [ ] Test and Release

**Please check any characteristics that apply to this pull request.**

- [x] Does not have any user-facing changes. This may include API changes, behavior changes, performance improvements, etc.

### CODE REVIEW

**Comment by liu-song** [comment]
这个错误貌似是common/model 和 common/utils 相互引用了，我再看下有没有什么好点的方式

File: common/time/time.go [hunk 1]
```diff
@@ -62,3 +62,13 @@ func (d *Duration) UnmarshalJSON(b []byte) error {
 		return errors.New("invalid duration")
 	}
 }
+
+// time.Time转为字符串时间
+func Time2String(t time.Time) string {
```
Comment (andrewshan) [inline common/time/time.go:67]: 1. 注释需要符合go的注释规范
2. 注释需要使用英文

File: healthcheck/server.go [hunk 2]
```diff
@@ -21,6 +21,7 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	time2 "github.com/polarismesh/polaris-server/common/time"
```
Comment (chuntaojun) [inline healthcheck/server.go:24]: time2 is recommended to use other names instead, such as commontime, etc.

Comment (liu-song) [reply healthcheck/server.go]: DONE

File: common/time/time.go (hunk 1 above)
Comment (liu-song) [reply common/time/time.go:67]: DONE

**Review by chuntaojun — APPROVED** [review]

**Pull request merged**